### DISCUSSION TITLE
How to disable all logging in config.json?

### DISCUSSION THREAD

**Discussion by slimy-pufflefish** [bghira/SimpleTuner#1177] [Q&A] [answered]
Is there a way to disable all logging in config.json?

I tried adding `"--report_to": ""` in the config.json, but that did not work, because I'm required to specify at least one logging method.

**Comment by bghira** [comment, marked as answer]
can you not open a discussion for everrrryy single question? there is a document called OPTIONS that has this stuff in it.

**Reply by slimy-pufflefish** [reply]
Thanks for the heads-up -- very helpful! And yeah, I'll be sure to consult the docs in the future. Thanks for great work on SimpleTuner :)
